macros/parsers/parserGraphTool.pl

```diff
@@ -762,24 +762,23 @@ sub constructJSXGraphOptions {
 # to generate a printable graph instead.  An attempt is made to make the printable graph look
 # as much as possible like the JavaScript graph.
 sub ans_rule {
-	my $self = shift;
-	my $out  = main::NAMED_HIDDEN_ANS_RULE($self->ANS_NAME);
+	my $self     = shift;
+	my $ans_name = $self->ANS_NAME;
```

Comment:
To fix the problem with the answer not being recorded you could change this to
```suggestion
	my $answer_value = $main::envir{inputs_ref}{ $self->ANS_NAME } // '';
	my $ans_name     = main::RECORD_ANS_NAME($self->ANS_NAME, $answer_value);
```
and then change line 923 as suggested below.  None of the other processing done by NAMED_HIDDEN_ANS_RULE is needed.